### PR TITLE
added some emoji-keywords!

### PR DESCRIPTION
**Pull request by tanishq-soni**

### CODE REVIEW

File: lib/emojiKeywords.json [hunk 1]
```diff
@@ -83,5 +83,8 @@
   "react": "react",
   "apple": "appleinc",
   "cat": "cat",
-  "dog": "dog"
+  "dog": "dog",
+  "code": "male-technologist",
+  "hack": "male-technologist",
```
Comment (sampoder) [inline lib/emojiKeywords.json:88]: could we make this the non-binary version of the emoji?

File: lib/emojiKeywords.json [hunk 2]
```diff
@@ -83,5 +83,8 @@
...
+  "autumn": "hackautumn"
```
Comment (sampoder) [inline lib/emojiKeywords.json:89]: rn all of autumn posts are tagged with :aom: could we continue that trend?

File: lib/emojiKeywords.json (hunk 1 above)
Comment (sampoder) [reply lib/emojiKeywords.json:88]: referring to https://emojipedia.org/technologist/, though im not sure if Slack supports it

if not :goose-honk-technologist: maybe?

File: lib/emojiKeywords.json (hunk 2 above)
Comment (tanishq-soni) [reply lib/emojiKeywords.json:89]: btw i created :aom: and i liked :hackautum: more than that :/

File: lib/emojiKeywords.json (hunk 1 above)
Comment (tanishq-soni) [reply lib/emojiKeywords.json:88]: sure :goose-honk-technologist: is nice! 👍🏻

File: lib/emojiKeywords.json (hunk 2 above)
Comment (tanishq-soni) [reply lib/emojiKeywords.json:89]: should i remove it?

File: lib/emojiKeywords.json (hunk 1 above)
Comment (tanishq-soni) [reply lib/emojiKeywords.json]: check it now 🧑‍💻 on slack using ":technologist:"

Comment (tanishq-soni) [reply lib/emojiKeywords.json]: check it now 🧑‍💻 "technologist" on slack

File: lib/emojiKeywords.json (hunk 2 above)
Comment (MatthewStanciu) [reply lib/emojiKeywords.json:89]: I think hackautumn is fine. Scrappy reacts with aom by default

**Pull request merged**